tests/unit/remote/ssh/test_connection.py

```diff
@@ -0,0 +1,46 @@
+import mockssh
+import os
+from pytest import yield_fixture
+
+from dvc.remote.ssh import RemoteSSH
+
+__here__ = os.path.abspath(os.path.dirname(__file__))
```

Comment:
It is not a good idea to use `__name__` for your regular variables.

Reply:
Yup :)